css/main.css

```diff
@@ -1,55 +1,127 @@
-body{
-    margin: 0;
-    padding: 0;
+/* applies to all in HTML body */
+body {
     font-family: sans-serif;
     color: white;
     background: #2d3436;
 }
 
+/* ------------------------------- */
+/* MULTI USE PARAMETERS START */
+/* ------------------------------- */
+/* sapces label front input fields */

```

Comment:
`/* spaces label front input fields */`
-Fix Typo

Reply:
The inconsistent spacing at the end bothers me